### PR TITLE
Some docs updates.

### PR DESCRIPTION
**Pull request by kasenvr**

### CODE REVIEW

**Review by ctrlaltdavid — APPROVED** [review]

File: docs/source/create.rst [hunk 1]
```diff
@@ -2,9 +2,9 @@
 Create 
 ######
 
-Vircadia enables people connected by interest, community, and friendship to come together and express their creativity with each other. We invite you to personalize your own experience by creating avatars and wearables, building immersive experiences, and developing apps to make the metaverse your own. 
+Vircadia enables people connected by interest, community, and friendship to come together and express their creativity with each other. We invite you to personalize your own experience by creating avatars, content, and developing apps/scripts. To truly make the metaverse your own, creating your own world or building your immersive project using the free and openly licensed Vircadia engine is encouraged!
```
Comment (JulianGro) [inline docs/source/create.rst:5]:
```suggestion
Vircadia enables people connected by interest, community, and friendship to come together and express their creativity with each other. We invite you to personalize your own experience by creating avatars and content, and developing apps and scripts. To truly make the metaverse your own, creating your own world or building your immersive project using the free and openly licensed Vircadia engine is encouraged!
```
Might be a lot of "and"s, but at least it should be grammatically correct.

File: docs/source/create/environments.rst [hunk 2]
```diff
@@ -19,10 +19,10 @@ An environment consists different types of 3D models and a skybox following the
 |                        | <3d-models/import-model.html#import-3d-models-using-the-asset-server>`_.                            |
 +------------------------+-----------------------------------------------------------------------------------------------------+
 | Your 3D Models         | You can use 3D modeling software like Blender or Maya to create your own 3D models and              |
-|                        | `import them into your Athena domain                                                                |
+|                        | `import them into your Vircadia domain                                                              |
```
Comment (JulianGro) [inline docs/source/create/environments.rst:22]:
```suggestion
|                        | `import them into your Vircadia Domain                                                              |
```

File: docs/source/create/environments.rst [hunk 3]
```diff
@@ -19,10 +19,10 @@ An environment consists different types of 3D models and a skybox following the
...
 |                        | <3d-models/import-model.html#import-3d-models-using-the-asset-server>`_.                            |
 +------------------------+-----------------------------------------------------------------------------------------------------+
-| A Content Archive File | Athena domain settings have downloadable content archives. These archives are zip                   |
+| A Content Archive File | Vircadia domain settings have downloadable content archives. These archives are zip                 |
```
Comment (JulianGro) [inline docs/source/create/environments.rst:25]:
```suggestion
| A Content Archive File | Vircadia Domain settings have downloadable content archives. These archives are zip                 |
```

File: docs/source/faq.rst [hunk 4]
```diff
@@ -38,19 +38,13 @@ Domain Hosting
...
-|                                                                                                     |
-|     * **Windows**: %AppData%/Roaming/Project Athena/domain-server                                   |
-|     * **Mac**: ~/Library/Application Support/Project Athena/domain-server                           |
+| The authentication settings for your domain are encrypted into a configuration                      |
+| file on the server's file system. To 'reset' the authentication settings, you need to               |
+| manually remove the HTTP security settings ('http_password' and 'http_username') from               |
+| config.json. The config file is stored on the server in the following directories:                  |
+|                                                                                                     |
+| * **Windows**: %AppData%/Roaming/Vircadia/domain-server                                             |
+| * **Mac**: ~/Library/Application Support/Vircadia/domain-server                                     |
```
Comment (JulianGro) [inline docs/source/faq.rst:47]:
```suggestion
| * **Mac**: ~/Library/Application Support/Vircadia/domain-server/                                    |
| * **Linux**: /var/lib/vircadia/default/domain-server/                                               |
```

File: docs/source/faq.rst [hunk 5]
```diff
@@ -38,19 +38,13 @@ Domain Hosting
...
-|   config.json. The config file is stored on the server in the following directories:                |
-|                                                                                                     |
-|     * **Windows**: %AppData%/Roaming/Project Athena/domain-server                                   |
-|     * **Mac**: ~/Library/Application Support/Project Athena/domain-server                           |
+| The authentication settings for your domain are encrypted into a configuration                      |
+| file on the server's file system. To 'reset' the authentication settings, you need to               |
+| manually remove the HTTP security settings ('http_password' and 'http_username') from               |
+| config.json. The config file is stored on the server in the following directories:                  |
+|                                                                                                     |
+| * **Windows**: %AppData%/Roaming/Vircadia/domain-server                                             |
```
Comment (JulianGro) [inline docs/source/faq.rst:46]:
```suggestion
| * **Windows**: %AppData%\Roaming\Vircadia\domain-server\                                            |
```

**Review by JulianGro — CHANGES_REQUESTED** [review]
There is a good amount of outdated stuff around, which I didn't comment on. This is likely worth a bigger sweep later.

**Review by JulianGro — APPROVED** [review]

**Pull request merged**